<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>261</height>
      </rect>
     </property>
     <attribute name="label">
      <string>用户信息管理</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>查询用户</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sendMessage_function">
         <property name="text">
          <string>发送通知</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>261</height>
      </rect>
     </property>
     <attribute name="label">
      <string>包裹信息管理</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>111</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_queryPackage_function">
         <property name="text">
          <string>查询包裹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_alterPakege_fuction">
         <property name="text">
          <string>修改包裹信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_instorge_fuction">
         <property name="text">
          <string>入库记录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_outstorge_function">
         <property name="text">
          <string>出库记录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>261</height>
      </rect>
     </property>
     <attribute name="label">
      <string>订单信息</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>95</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_queryOrder_fuction">
         <property name="text">
          <string>订单查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_processRequest_function">
         <property name="text">
          <string>待处理订单</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>261</height>
      </rect>
     </property>
     <attribute name="label">
      <string>包裹操作</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_addInstore">
         <property name="text">
          <string>包裹入库</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sendPakage">
         <property name="text">
          <string>包裹发出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>1071</width>
      <height>691</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_4">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>1041</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>信息一览</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>入库记录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>包裹数量</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>订单数量</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>50</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>出库记录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5"/>
   </widget>
   <widget class="QPushButton" name="pushButton_manager">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>你好，管理员</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据库</string>
    </property>
    <addaction name="actionadd_connection"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>账号信息</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>统计数据</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionadd_connection">
   <property name="text">
    <string>连接数据库</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
